GsaGH/Components/2_Geometry/CreateMember2d.cs

```diff
@@ -110,19 +113,20 @@ public class CreateMember2d : GH_OasysComponent {
         "Ms",
         "Target mesh size",
         GH_ParamAccess.item);
+      pManager.AddBooleanParameter("Internal Offset",
+        "Io",
+        "Set Automatic Internal Offset of Member",
+        GH_ParamAccess.item);
```

Comment:
Suggest we add a default to the input so users can see what the default is when hovering over the input in Grasshopper:
```suggestion
        GH_ParamAccess.item,
        false);
```